docs/root/call-settings.md

```diff
@@ -58,3 +58,71 @@ for multiple RPC invocations.
 By default there is a per-RPC-method client configuration that specifies a sensible
 retry policy for every method. See, for example,
 [PublisherSettings.CreateTopicSettings](Google.Pubsub.V1/api/Google.Pubsub.V1.PublisherSettings.html#Google_Pubsub_V1_PublisherSettings_CreateTopicSettings)
+
+## Timing settings
+
+The timing settings for an RPC are specified using [CallTiming](obj/api/Google.Api.Gax.Grpc.CallTiming.yml).
+This is basically either a simple [Expiration](obj/api/Google.Api.Gax.Grpc.Expiration.yml),
+or a [RetrySettings](obj/api/Google.Api.Gax.Grpc.RetrySettings.yml). `RetrySettings` uses an `Expiration` as well,
+so let's look at `Expiration` first.
+
+### Expiration
+
+An expiration specifies the latest point at which you're willing for an operation to complete. It can be either
+a *timeout* (relative to the start time, e.g. "it can't take more than 5 seconds") or a *deadline* (which is
+absolute, e.g. "it must finish by 2016-12-07 14:55:00Z"). You create an `Expiration` by specifying that timeout
+or deadline, with the `Expiration.FromTimeout(TimeSpan)` and `Expiration.FromDeadline(DateTime)` methods.
+Alternatively, you can use `Expiration.None`, which is an infinite timeout.
+
+Note that deadline expirations are most commonly used when the same deadline is passed to multiple calls:
+if you have to perform 5 different RPCs, and you know your total work needs to finish at a particular
+point but you don't care how those 5 RPCs use up your time budget between them, passing in the same
+deadline to all 5 works well. This is restricted to one unit of work, however - you'd need to calculate a new
+deadline for the next unit of work, whereas a relative timeout may be valid for all calls. You should think
```

Comment:
Yes, it's relative to the start time of the RPC.
This should probably be made clearer in this doc.